### PR TITLE
Fix: Managers of a parent project should be able to edit request reports

### PR DESCRIPTION
**Pull request by ludwiktrammer**
Workaround for https://github.com/odoo/odoo/issues/5121

### CODE REVIEW

File: addons/bestja_requests/models.py [hunk 1]
```diff
@@ -111,6 +115,49 @@ class RequestItem(models.Model):
...
+        # See also: https://github.com/odoo/odoo/issues/5121
+        #
+        # 3. That's a problem for us, because `self.request.project` in some cases
+        # may not be accessible by the current user (if she's a coordinator of a
+        # parent organization). And we need to get through `self.request.project`
+        # to get to `request_template`.
+        #
+        # ---
+        #
+        # You should also note the impotence of the exact placement of sudo() in
```
Comment (KrzysiekJ) [inline addons/bestja_requests/models.py]: I guess it should be the “importance”. ;)

File: addons/bestja_requests/models.py [hunk 2]
```diff
@@ -111,6 +115,49 @@ class RequestItem(models.Model):
...
+        # the code below. You can not use sudo() with records in draft mode, because
+        # they only exist in the current environment, and sudo() creates a new
+        # environment. In this case this means that you shouldn't put sudo()
+        # directly after `self` or `request`. You can put sudo() after `project`,
+        # which is not in draft mode, even though it may not be accessible to the
+        # current user, because the read will be performed only when we try
+        # to access one of its fields (`parent` in this particular case).
+        #
+        # That's also why we can't just use the `request.parent_project` related field,
+        # even though it is stored and have a precomputed value already. Stored
```
Comment (KrzysiekJ) [inline addons/bestja_requests/models.py]: “has”

**Comment by ludwiktrammer** [comment]
Done.

**Comment by KrzysiekJ** [comment]
Merged in 483d4730e470c643c06f07779d618cd140a71b6a (there was a conflict).

**Pull request closed without merging**